packages/calypso-products/src/plans-list.tsx

```diff
@@ -1651,6 +1651,7 @@ PLANS_LIST[ PLAN_WPCOM_STARTER ] = {
 		FEATURE_PAYMENT_BLOCKS,
 		FEATURE_TITAN_EMAIL,
 	],
+	getIncludedFeatures: () => [ WPCOM_FEATURES_ATOMIC, WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS ],
```

Comment:
I guess we should replicate the same changes from D83396-code and move the Atomic feature to the Marketplace addons

Reply:
Since the Marketplace products are not currently listed in this file, another option would be to leave that for a follow-up and perform "is Marketplace product" checks in the meantime (like what we have done in https://github.com/Automattic/wp-calypso/pull/65215 or https://github.com/Automattic/wp-calypso/pull/65191).